modules/farm_fd2/src/entrypoints/direct_seeding/App.vue

```diff
@@ -61,9 +61,6 @@
           v-bind:pickedBeds="form.beds"
           v-bind:showValidityStyling="validity.show"
           v-on:valid="validity.location = $event"
-          v-on:update:beds="
```

Comment:
Still need this call here, but need to just set `form.beds` in the function `handleBedsUpdate`.